### PR TITLE
demo(form): label set to null

### PR DESCRIPTION
**Pull request by thinkasany**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [ ] 📝 Site / documentation improvement
- [x] 📽️ Demo improvement

### 🔗 Related Issues

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      demo(form): label set to null     |
| 🇨🇳 Chinese |     demo(form): 设置lebel为null    |

### CODE REVIEW

File: components/form/demo/validate-scroll-to-field.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const App = () => {
       labelCol={{ span: 6 }}
       wrapperCol={{ span: 14 }}
     >
-      <Form.Item wrapperCol={{ offset: 6 }}>
+      <Form.Item label={null}>
```
[the remark this answers is not in the document]
Comment (afc163) [reply components/form/demo/validate-scroll-to-field.tsx:15]: 先不改吧，如果 antd 没升级上来，这么写没效果。

[the remark this answers is not in the document]
Comment (crazyair) [reply components/form/demo/validate-scroll-to-field.tsx:15]: 可以各几个版本后改，比如 Flex 组件，也是之后替换的

[the remark this answers is not in the document]
Comment (thinkasany) [reply components/form/demo/validate-scroll-to-field.tsx:15]: @afc163 其实打开form的第一个demo就已经这么推荐写了，考虑到 ```"antd 没升级上来，这么写没效果, 隔几个版本后改"```, 是不是合到feature分支就刚好了。
image

[the remark this answers is not in the document]
Comment (afc163) [reply components/form/demo/validate-scroll-to-field.tsx:15]: 我倾向至少等 3 个月。

[the remark this answers is not in the document]
Comment (crazyair) [reply components/form/demo/validate-scroll-to-field.tsx:15]: 这个功能比较隐晦，不是 api 直接提示不存在，这个在旧版本也也不会报错，导致用户都这样用了

**Pull request closed without merging**